bst_plugin/process_nst_OM_from_cortex.m

```diff
@@ -55,5 +55,5 @@
 [head_vertex_ids, sHead] = process_nst_cpt_fluences_from_cortex('proj_cortex_scout_to_scalp', ...
                                                                  cortex_scout_selection, ...
                                                                  sProcess.options.cortex_to_scalp_extent.Value{1}.*0.01, 1);
-OutputFiles = process_nst_OM_from_head('Compute', sProcess, cortex_scout_selection.sSubject, sHead, head_vertex_ids);
+OutputFiles = process_nst_OM_from_head('Compute', sProcess, cortex_scout_selection.sSubject, sHead, head_vertex_ids, sInputs);
```

Comment:
current `process_nst_OM_from_head`  does not have sInputs as last arg.
Maybe yu have some uncommited changes to  `process_nst_OM_from_head`